hs_core/HydroRealtimeSignalProcessor.py

```diff
@@ -0,0 +1,110 @@
+from django.db import models
+from haystack.signals import RealtimeSignalProcessor
+from haystack.exceptions import NotHandled
+import logging
+import types
+
+logger = logging.getLogger(__name__)
+logger.debug("load real time processor")
```

Comment:
@xuezhaokun I think we should remove these logging debug messages for production for performance reasons. You can always added them back in when you need to debug, but we should not include these messages in production.